Source/Diagnostics/WarpXIO.cpp

```diff
@@ -90,7 +90,7 @@ WarpX::InitFromCheckpoint ()
             std::istringstream lis(line);
             int i = 0;
             while (lis >> word) {
-                t_new[i++] = std::stod(word);
+                t_new[i++] = static_cast<Real>(std::stod(word));
```

Comment:
Just ping-ing @WeiqunZhang to take a look at this file, too - because I am not too familiar with it.

Reply:
I've just added @WeiqunZhang to the reviewers list

Reply:
Seems busy, but I think this is fine :)